pom.xml

```diff
@@ -354,6 +354,9 @@
         <test.included.category>
           org.jboss.hal.testsuite.category.Elytron
         </test.included.category>
+        <test.excluded.category>
```

Comment:
We need to be able to choose if we want to run KnownIssue tests or not. This way of implementation excludes KnownIssues tests every time. 
We rather need new profile which let us choose if we want to run them or not.